.azure_pipelines/ci-pipeline-dotnet-test-suite.yml

```diff
@@ -0,0 +1,66 @@
+variables:
+  - group: myst-pipeline
+  - name: SCRIPTS_ROOT
+    value: $(Build.SourcesDirectory)/.azure_pipelines/scripts
+
+# dotnet tests pipeline will be triggered in pipeline schedule settings.
+trigger: none
+pr: none
+
+jobs:
+  - job: BuildAndTest
```

Comment:
Theres an extra space here before the '-' that az pipelines might complain about

Reply:
The entire thing would accordingly go forward by 1 space.. (this is the syntax mistake I always make too)

Reply:
The indentation here is similar to ci-pipelines-makefile.yml, which indents the jobs list by 2 spaces.

Reply:
Gotcha, I was looking at [this](https://github.com/deislabs/mystikos/blob/main/.azure_pipelines/ci-pipeline-makefile-nightly.yml). Should work then.